ts/test/test-init-config.ts

```diff
@@ -66,7 +66,10 @@ describe('initConfig', () => {
       timeIntervalMicros: 1000,
       heapIntervalBytes: 512 * 1024,
       heapMaxStackDepth: 64,
-      backoffMillis: 300000
+      initialBackoffMillis: 1000,
+      maxBackoffMillis: 60 * 60 * 1000,
+      backoffMultiplier: 1.3,
+      backoffLimitMillis: 7 * 24 * 60 * 60 * 1000
```

Comment:
All these values being duplicated.. is there a way to not have this config object duplicated everywhere? This adds to the burden to modifying or adding tests.

Reply:
I've simplified this -- done.